jobs/check-aws-regions.Jenkinsfile

```
@@ -0,0 +1,33 @@
+import org.yaml.snakeyaml.Yaml;
+
+node {
+    checkout scm
+    // these are script global vars
+    pipeutils = load("utils.groovy")
+}
+
+properties([
+    pipelineTriggers([
+        githubPush(),
+        // once a day
+        pollSCM('H H * * *')
+    ]),
+    buildDiscarder(logRotator(
+        numToKeepStr: '100',
+        artifactNumToKeepStr: '100'
+    )),
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+])
+
+cosaPod(serviceAccount: "jenkins"){
+    def disabled_regions = ""
+    withCredentials([file(variable: 'AWS_CONFIG_FILE',
+                           credentialsId: 'aws-build-upload-config')]) {
+        disabled_regions = shwrapCapture("ore aws list-regions --disabled")
+    }
+    if (disabled_regions != "None") {
+        echo "Disabled regions: ${disabled_regions}"
```

Maybe let's call the [`warn()`](https://github.com/coreos/coreos-ci-lib/blob/main/vars/warn.groovy) function to set the state of the job to yellow:

```suggestion
        
        return
```
```suggestion
        warn("Disabled AWS region detected: ${disabled_regions}")
```